SparkPost/2web2ui
# FE-1235 : Add View filters button and Modal to Saved Reports Section

FE-1235 : Add View filters button and Modal to Saved Reports Section

_Based off of FE-1266_ 

### What Changed

- Added View Filters Button in the Aggregate Metrics Section
- This report should show only when there are any filters associated with the pinned report
- On Clicking the View Report Button in this Modal, it should take you to the report Builder page
- Added the PinnedReportFilterModal that opens on clicking the View Filters Button
- Moved the ActiveFilters component to src/components/ReportBuilder 

### How To Test

- Add the View Filters button to the metrics section. -  https://sparkpost.invisionapp.com/console/share/RU1SP41BG7
- Do not show “View filters” button if there are no filters associated with the pinned report.
- Clicking on View filter opens this Modal -  https://sparkpost.invisionapp.com/console/share/RU1SP41BG7/483505019
- Clicking on View Report, should redirect to Report Builder with the pinned report report options

### To Do

- [ ] Add related cypress tests and unit tests
- [x] Verify with UX if the Modal looks good or do they want the filters to have white background.
